POLICY.md

```diff
@@ -0,0 +1,41 @@
+# Policy
+
+This document will lay out the official policies used within Pulsar-Edit. This document lists the various policies that have come about naturally as the project progressed and so deserve to be properly documented and viewable by anyone.
```

Comment:
Maybe I am over-thinking this, but I feel like the word "official" here may lend itself to people wanting to bonk us for "not following our own rules!!1"

Maybe delete the first sentence "This document will lay out the official policies [ . . . ]"  in favor of just the second one "This document lists the various policies that have come about naturally [ . . . ]".

These things evolve, I'd rather be accountable to people than documents.

Reply:
I've just seen this, and while don't have the time to resolve it. Just wanted to say totally agree with the point your making, and I think for this at least it's a good idea to distance ourselves from "official"

Reply:
@DeeDeeG Added your changes, since they seemed great to add. Let me know what you think and if we are good to merge. (If you've got the chance, otherwise I can take the previous approval and go ahead)

Reply:
Yeah, the latest changes LGTM, and I noticed a couple typo fixes in there too, good spotting those.

👍

Reply:
Thanks for checking through it again! I'll go ahead and merge this one then